springboot/src/main/java/ca/jrvs/apps/trading/controller/TraderAccountController.java

```diff
@@ -0,0 +1,105 @@
+package ca.jrvs.apps.trading.controller;
+
+import ca.jrvs.apps.trading.model.domain.Account;
+import ca.jrvs.apps.trading.model.domain.Trader;
+import ca.jrvs.apps.trading.model.domain.TraderAccountView;
+import ca.jrvs.apps.trading.service.TraderAccountService;
+import io.swagger.annotations.Api;
+import java.sql.Date;
+import java.time.LocalDate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.format.annotation.DateTimeFormat.ISO;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Api(value = "Trader", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+@Controller
+@RequestMapping("/trader")
+public class TraderAccountController {
+  private TraderAccountService traderAccountService;
+
+  @Autowired
+  public TraderAccountController(
+      TraderAccountService traderAccountService) {
+    this.traderAccountService = traderAccountService;
+  }
+
+  @ResponseStatus(HttpStatus.CREATED)
+  @ResponseBody
+  @PostMapping(
+      path = "/firstname/{firstname}/lastname/{lastname}/dob/{dob}/country/{country}/email/{email}",
+      produces = {MediaType.APPLICATION_JSON_UTF8_VALUE})
+  public TraderAccountView createTrader(@PathVariable String firstname,
+      @PathVariable String lastname,
+      //@PathVariable @DateTimeFormat(pattern = "yyy-MM-dd") LocalDate dob,
+      @PathVariable @DateTimeFormat(iso = ISO.DATE) LocalDate dob,
+      @PathVariable String country,
+      @PathVariable String email){
+
+    try{
+      Trader trader = new Trader();
+      trader.setFirstName(firstname);
+      trader.setLastName(lastname);
+      trader.setCountry(country);
+      trader.setEmail(email);
+      trader.setDob(Date.valueOf(dob));
+
+      return traderAccountService.createTraderAccount(trader);
+    }catch(Exception e){
+      throw ResponseExceptionUtil.getResponseStatusException(e);
```

Comment:
please use IDE to auto format your code.

https://www.notion.so/jarviscanada/IntelliJ-IDEA-Guide-2e124ee013ee4ef3b8e1ee55e3b494aa#69ab340642404bfdb08cf5d9be36454e